events.html

```diff
@@ -5,9 +5,9 @@
 	<meta charset="UTF-8" />
 	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-	<meta property="og:title" content="Web Mini-Conference">
-	<meta property="og:description" content="Conference on important concepts a Web Developer should know about.">
-	<meta property="og:image" content="https://realdevsquad.com/img/events/og-image.png">
+	<meta property="og:title" content="Upcoming Events">
+	<meta property="og:description" content="Events on important concepts a Web Developer should know about.">
+	<!-- <meta property="og:image" content="https://realdevsquad.com/img/events/og-image.png"> --> <!--Don't have image for events-->
```

Comment:
Use the poster image for the event please

Reply:
Done